labs/apis/grid_lists/index.html

```diff
@@ -1,94 +0,0 @@
-<html>
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-  <title>ApeFlickz Top Picks</title>
-
-  <style>
-    @font-face {
-      font-family: Shnickers;
-      src: url(LUNCN___.TTF);
-    }
-  </style>
-  <script type="module" src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.esm.js"></script>
-  <script nomodule src="https://unpkg.com/ionicons@7.1.0/dist/ionicons/ionicons.js"></script>
-
-  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
-    integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
-  <link rel="stylesheet" href="style.css">
-</head>
-
-<body>
-  <div id="background">
-    <nav class="navbar navbar-expand-lg bg-danger">
-      <div class="container-fluid">
-        <a href="#" class="navbar-brand"><img src="Ape-Logo.png" alt="" style="width: 150px;"></a>
-        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarSupportedContent">
-          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
-            <li class="nav-item">
-              <a class="nav-link active" aria-current="page" href="#" style="font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;font-size: medium;color: white;">Home</a>
-            </li>
-          </ul>
-        </div>
-      </div>
-    </nav>
-    <div class="container pt-3">
-      
-      <br>
-      <br>
-      <h3 id="head-text" class="glow">ApeFlickz Top Movies</h3>
-      <br>
-      <table id="shows" class="table table-striped table-hover table-bordered"></table>
-    </div>
-
-    <br>
-  <br>
-  <br>
-  <br>
-  <footer>
-    <div class="shadow-lg border bg-danger border-dark border-bottom-0 border-start-0 border-end-0">
-      <br>
-     <div class="row">
-      <div class="col border border-0 bg-mute">

```

Comment:
Q-*/F-*: Your work doesn't appear in this PR.
 - Q-4: You have some odd spacing in your index.html.
 - F-1: This uses a table instead of a grid with rows and cols. This circumvents most of the purpose of the assignment.